tuned/plugins/plugin_cpu.py

```
@@ -814,7 +820,10 @@ def _get_boost(self, device, instance, ignore_missing=False):
 			return None
 		cpu_id = device.lstrip("cpu")
 		if os.path.exists(self._pstate_boost_path(cpu_id)):
-			return self._cmd.read_file(self._pstate_boost_path(cpu_id)).strip()
+			val = self._cmd.read_file(self._pstate_boost_path(cpu_id)).strip();
```

Ruff (0.15.12)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)